conf.yaml

```diff
@@ -1246,8 +1246,8 @@ contents:
                   - title:      APM Go Agent
                     prefix:     go
                     current:    1.x
-                    branches:   [ master, 1.x, 0.5 ]
-                    live:       [ master, 1.x ]
+                    branches:   [ {main: master}, 1.x, 0.5 ]
```

Comment:
what's this "main: master" for?

Reply:
This line maps the `main` branch of the`elastic/apm-agent-go` repository to the [`master` documentation URL](https://www.elastic.co/guide/en/apm/agent/go/master/index.html). I believe we originally did this because it was the easiest way to incrementally move documentation to `main`, without breaking the doc build. Once all Elastic documentation repositories are on `main`, we can turn this mapping off, publish docs to the `main` documentation URL instead, and turn on one huge redirect from `master` to `main` docs.

Reply:
Makes sense, thanks!